packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1078,6 +1078,13 @@ async function handler(
     if (!originalRescheduledBooking) {
       throw new HttpError({ statusCode: 404, message: "Could not find original booking" });
     }
+
+    if (
+      originalRescheduledBooking.status === BookingStatus.CANCELLED &&
+      !originalRescheduledBooking.rescheduled
```

Comment:
when organizer selects "request reschedule" then reschedule is set to true and that booking can be rescheduled